qabelbox/src/androidTest/java/de/qabel/qabelbox/repositories/SqliteContactRepositoryTest.kt

```diff
@@ -0,0 +1,309 @@
+package de.qabel.qabelbox.repositories
+
+import android.support.test.InstrumentationRegistry
+import android.support.test.runner.AndroidJUnit4
+import android.test.RenamingDelegatingContext
+import de.qabel.core.config.Contact
+import de.qabel.core.config.Identity
+import de.qabel.core.config.factory.DropUrlGenerator
+import de.qabel.core.config.factory.IdentityBuilder
+import de.qabel.core.crypto.QblECPublicKey
+import de.qabel.core.repository.ContactRepository
+import de.qabel.core.repository.EntityManager
+import de.qabel.core.repository.IdentityRepository
+import de.qabel.core.repository.exception.EntityExistsException
+import de.qabel.core.repository.exception.EntityNotFoundException
+import de.qabel.qabelbox.persistence.RepositoryFactory
+import org.hamcrest.Matchers.*
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.*
+
+@RunWith(AndroidJUnit4::class)
+class SqliteContactRepositoryTest {
```

Comment:
Because it needs to be run on the android device itself because that one uses a different jdbc driver.